lib/zip/entry.rb

```diff
@@ -83,7 +83,7 @@ def time
       end
     end
 
-    alias :mtime :time
+    alias_method :mtime, :time
```

Comment:
Yeah, agree with you based on https://github.com/bbatsov/ruby-style-guide#alias-method, I redone it soon in all cases